scripts/install.sh

```diff
@@ -1,116 +1,102 @@
 #!/bin/bash
-set -euo pipefail
+set -eo pipefail
 
 LOG_START='\n\e[1;36m' # new line + bold + color
 LOG_END='\n\e[0m' # new line + reset color
 DONE_START='\n\e[1;32m' # new line + bold + green
 DONE_END='\n\n\e[0m'    # new line + reset
 
 KEEP_CORE_PATH=$PWD
-KEEP_CORE_SOL_PATH="$KEEP_CORE_PATH/solidity-v1"
+
+KEEP_BEACON_SOL_PATH="$KEEP_CORE_PATH/solidity/random-beacon"
 
 # Defaults, can be overwritten by env variables/input parameters
-CONFIG_DIR_PATH_DEFAULT="$KEEP_CORE_PATH/configs"
-NETWORK_DEFAULT="local"
+NETWORK_DEFAULT="development"
 KEEP_ETHEREUM_PASSWORD=${KEEP_ETHEREUM_PASSWORD:-"password"}
-CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY=${CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY:-""}
 
 help()
 {
    echo -e "\nUsage: ENV_VAR(S) $0"\
-           "--config-dir <path>"\
            "--network <network>"\
-           "--contracts-only"
+           "--skip-deployment"\
+           "--skip-client-build"
    echo -e "\nEnvironment variables:\n"
    echo -e "\tKEEP_ETHEREUM_PASSWORD: The password to unlock local Ethereum accounts to set up delegations."\
            "Required only for 'local' network. Default value is 'password'"
-   echo -e "\tCONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY: Contracts owner private key on Ethereum. Required for non-local network only"
    echo -e "\nCommand line arguments:\n"
-   echo -e "\t--config-dir: Path to keep-core client configuration file(s)"
    echo -e "\t--network: Ethereum network for keep-core client."\
-                        "Available networks and settings are specified in the 'truffle-config.js'"
-   echo -e "\t--contracts-only: Should execute contracts part only."\
-                        "Client installation will not be executed.\n"
+                        "Available networks and settings are specified in the 'hardhat.config.ts'"
+   echo -e "\t--skip-deployment: When set to true the old artifacts from the '/deployments' dir are used. Default is false"
+   echo -e "\t--skip-client-build: Should execute contracts part only. Client installation will not be executed\n"
    exit 1 # Exit script after printing help
 }
 
 # Transform long options to short ones
 for arg in "$@"; do
   shift
   case "$arg" in
-    "--config-dir")     set -- "$@" "-c" ;;
-    "--network")        set -- "$@" "-n" ;;
-    "--contracts-only") set -- "$@" "-m" ;;
-    "--help")           set -- "$@" "-h" ;;
-    *)                  set -- "$@" "$arg"
+    "--network")           set -- "$@" "-n" ;;
+    "--stake-owner")       set -- "$@" "-o" ;;
+    "--staking-provider")  set -- "$@" "-p" ;;
```

Comment:
We don't need these options anymore in the install script, right?

Reply:
nope